assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -222,29 +222,29 @@ che.oauth.openshift.oauth_endpoint= NULL
 # OpenShift OAuth verification token URL.
 che.oauth.openshift.verify_token_url= NULL
 
-# Configuration of Bitbucket Server OAuth1 client. Used to obtain Personal access tokens.
+# Configuration of Bitbucket Server OAuth1 client. Used to obtain personal access tokens.
 # Location of the file with Bitbucket Server application consumer key (equivalent to a username).
 che.oauth1.bitbucket.consumerkeypath=NULL
-# Location of the file with Bitbucket Server application private key
+# Location of the file with Bitbucket Server application private key.
 che.oauth1.bitbucket.privatekeypath=NULL
-# Bitbucket Server URL. To work correctly with factories the same URL
+# Bitbucket Server URL. To work correctly with factories, the same URL
 # has to be part of `che.integration.bitbucket.server_endpoints` too.
 che.oauth1.bitbucket.endpoint=NULL
 
 
 ### Internal
 
 # {prod-short} extensions can be scheduled executions on a time basis.
-# This configures the size of the thread pool allocated to extensions that are launched on
+# This value configures the size of the thread pool allocated to extensions that are launched on
 # a recurring schedule.
 schedule.core_pool_size=10
 
-# DB initialization and migration configuration
-# If true, ignore scripts up to the version configured by baseline.version.
+# Database initialization and migration configuration.
+# If set to `true`, it ignores scripts up to the version configured by `baseline.version`.
```

Comment:
what is "it" in this sentence? Might want to ask @tolusha or @skabashnyuk what's happening here...

Reply:
`It` means DB Initialization component on che-server side.